src/App.tsx

```diff
@@ -10,9 +12,18 @@
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/" element={<HomePage />} />
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
+
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path=":personHref" element={<PeoplePage />} />
+            </Route>
```

Comment:
In React Router v6, nested routes should be defined within the parent component using the `Outlet` component. Please restructure the `PeoplePage` component to include an `Outlet` for rendering child routes.